framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Hmm well then. I'll stay neutral to this PR. If there are others approving this PR I'll go with it. But I'm not convinced.